config/packages.yaml

```diff
@@ -145,6 +145,13 @@ packages:
     target: []
     providers: {}
     compiler: []
+  dd4hep:
+    variants: +lcio+xercesc+hepmc3+geant4
```

Comment:
```suggestion
    variants: +lcio+xercesc+hepmc3+geant4+edm4hep
```

Reply:
I set it to be enabled  by default, but yes, it's good to add it explicitly